openownership/data-standard
# Increment version across the repo, closes #419

# Overview

- **What does this pull request do?**

This PR:

- Updates all example and test BODS json data to increment bodsVersion value
- Updates the version field value in all schema .json files
- Updates docs/conf.py and docs/index.rst to reference the latest version


- **How can a reviewer test or examine your changes?**

Read through conf.py to check that the changes make sense. Build the branch's docs locally to check it all looks/works ok. Search for remaining references to v0.2 (or v0.1). 

- Who is best placed to review it?

@ScatteredInk has done this release update work previously so is in a good position to check the work. But if @ScatteredInk isn't about then @siwhitehouse would be able to do this.


(Closes/Relates to) issue: #419 

## Translations

n/a

## Documentation & Release

n/a
